crates/common/mqtt_channel/src/topics.rs

```diff
@@ -157,6 +157,16 @@ impl TryInto<TopicFilter> for &str {
     }
 }
 
+impl From<Vec<TopicFilter>> for TopicFilter {
```

Comment:
It could be made more generic by implementing `FromIterator`, which would enable `collect()`ing a `TopicFilter` from any `Iterator<Item = TopicFilter>`, but can be done later.

Reply:
Done by https://github.com/thin-edge/thin-edge.io/pull/2108/commits/cfa53ead89d36fa244f9866477ecf7e57a7b8708

Used here: https://github.com/thin-edge/thin-edge.io/pull/2108/commits/52f657240a4b8b68c54d2bfc1cef35535d849507